packages/client/src/devices/InputMediaDeviceManager.ts

```diff
@@ -18,11 +21,7 @@ export abstract class InputMediaDeviceManager<
   /**
    * @internal
    */
-  enablePromise?: Promise<void>;
-  /**
-   * @internal
-   */
-  disablePromise?: Promise<void>;
+  statusChangePromise?: Promise<void>;
```

Comment:
Store enable/disable promise in a single value. We can use `this.state.pendingStatus` to differentiate between the two, but usually that is not necessary.